.github/workflows/prbuild.yaml

```diff
@@ -8,13 +8,31 @@ env:
 jobs:
   integration:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # use stable kubernetes_version values since they're included
```

Comment:
pretty slick, nice!

Reply:
:+1: I think this is the best of both worlds, thanks for pointing me in this direction. Now if only the jobs will run...